var/spack/repos/builtin/packages/py-bluepyemodelnexus/package.py

```diff
@@ -0,0 +1,24 @@
+# Copyright 2013-2023 Lawrence Livermore National Security, LLC and other
+# Spack Project Developers. See the top-level COPYRIGHT file for details.
+#
+# SPDX-License-Identifier: (Apache-2.0 OR MIT)
+
+from spack.package import *
+
+
+class PyBluepyemodelnexus(PythonPackage):
+    """Python library to optimize and evaluate electrical models (Nexus addon)."""
+
+    homepage = "https://bbpgitlab.epfl.ch/cells/bluepyemodelnexus"
+    git = "ssh://git@bbpgitlab.epfl.ch/cells/bluepyemodelnexus.git"
+
+    version("0.0.2", tag="bluepyemodelnexus-v0.0.2.dev21")
```

Comment:
I'm curious what the intention is when v0.0.2.dev22 gets created. I'm very much not a fan of pointing 0.0.2 to a new tag, because once you do that you don't know what an installed 0.0.2 means any more (was it dev21? dev22?).

Perhaps this could be 0.0.2.21? Alternatively, a 0.0.2 release could be made.

Reply:
I agree, we can definitely make a release